wikimedia/wikipedia-ios
# Native Editor - Implement cursor move buttons

**Phabricator:**
https://phabricator.wikimedia.org/T348087

### Notes
This PR adds logic to move the cursor around from the expanding menu buttons.

### Test Steps
1. Open article editor on Staging.
2. Tap the arrow on the expanding menu.
3. Tap the chevron buttons < > ^ v
4. Confirm cursor moves as it did in the previous editor.
